wgengine/magicsock/magicsock.go

```diff
@@ -1145,6 +1147,16 @@ func (c *Conn) sendUDP(ipp netip.AddrPort, b []byte) (sent bool, err error) {
 	} else {
 		if sent {
 			metricSendUDP.Add(1)
+
+			// TODO(kradalby): Do we need error variants of these?
```

Comment:
magicsock exports a number of error metrics, perhaps we should plumb these through to the `tailscaled_outbound_dropped_packets_total` metric? E.g.

https://github.com/tailscale/tailscale/blob/7dcf65a10a02740713f20c744cfd3ed38c4919d7/wgengine/magicsock/magicsock.go#L2916-L2918